libraries/core_libs/storage/src/storage.cpp

```diff
@@ -724,8 +724,8 @@ void DbStorage::addPbftCertVotedBlockToBatch(PbftBlock const& pbft_block, Batch&
 
 std::optional<PbftBlock> DbStorage::getPbftBlock(blk_hash_t const& hash) {
   auto res = getPeriodFromPbftHash(hash);
-  if (res.first) {
-    return getPbftBlock(res.second);
+  if (res) {
+    return getPbftBlock(res.value());
```

Comment:
if you already checking for ret has value, then accessing value is faster with *.
https://en.cppreference.com/w/cpp/utility/optional/value#Notes